UnitySDK/Assets/ML-Agents/Scripts/CommunicatorObjects/UnityToExternalGrpc.cs

```diff
@@ -2,10 +2,7 @@
 //     Generated by the protocol buffer compiler.  DO NOT EDIT!
 //     source: mlagents/envs/communicator_objects/unity_to_external.proto
 // </auto-generated>
-
-# if UNITY_EDITOR || UNITY_STANDALONE_WIN || UNITY_STANDALONE_OSX || UNITY_STANDALONE_LINUX
```

Comment:
NOPE, we need that line

Reply:
Was that line edited after it was generated by the protobuf compiler?